File: 02 Digits and Numbers - Foundation Programs/program-05.cpp
// Write a program that takes a 3-digit number as input and calculates the sum of all its digits, then displays
// the result.

#include<iostream>
using namespace std;
int main()
{
    int number,first_digit,middle_digit,last_digit,sum;
    cout<<"Enter 3-digit number : ";
    cin>>number;
    first_digit=number/100;
    middle_digit=(number/10)%10;
    last_digit=number%10;
    sum=first_digit+middle_digit+last_digit;

    cout<<"The sum of all its digits : "<<sum;

    return 0;
}

File: archit_vishwakarma/Level-1/03-Digits-And-Numbers/program-05.py
n = input("Enter a number : ")

try:
    n = int(n)
    if n < 1000 and n > 99:
        
        a = n%10
        b = (n//10)%10
        c = (n//100)%10

        print(f"Total of digits in number : {a+b+c}")
        
    else:
        print("Enter a Three Digit Number")
except ValueError:
    print("enter a valid number")